server/migrations/20210203200024-add-title-to-chain-entities.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'ChainEntities',
+          'title',
+          {
+            type: Sequelize.TEXT,
+            allowNull: true,
+          },
+          { transaction: t }
+        ),
+        queryInterface.addColumn(
+          'ChainEntities',
+          'author',
+          {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          { transaction: t }
+        )
+      ]);
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.removeColumn(
+          'ChainEntities',
+          'title',
+          {
```

Comment:
I think you need to remove the column specification for these removeColumn commands.

Reply:
Done!